packages-exp/auth-exp/src/platform_cordova/popup_redirect/popup_redirect.test.ts

```diff
@@ -85,6 +93,180 @@ describe('platform_cordova/popup_redirect/popup_redirect', () => {
     });
   });
 
+  describe('_initialize', () => {
+    const NO_EVENT_TIMER_ID = 10001;
+    const PACKAGE_NAME = 'my.package';
+    const NOT_PACKAGE_NAME = 'not.my.package';
+    let universalLinksCb:
+      | ((eventData: Record<string, string> | null) => unknown)
+      | null;
+    let tripNoEventTimer: TimerTripFn;
+
+    beforeEach(() => {
+      tripNoEventTimer = stubSingleTimeout(NO_EVENT_TIMER_ID);
+      window.universalLinks = {
+        subscribe(_unused, cb) {
+          universalLinksCb = cb;
+        }
+      };
+      window.BuildInfo = {
+        packageName: PACKAGE_NAME,
+        displayName: ''
+      };
+      sinon.stub(window, 'clearTimeout');
+    });
+
+    afterEach(() => {
+      universalLinksCb = null;
```

Comment:
Please clean up `universalLinks`, `BuildInfo` and `sinon.restore`.

Reply:
sinon.restore is done in the top-level afterEach(). Added the others